epub33/explainers/EPUB33-explainer.md

```diff
@@ -0,0 +1,134 @@
+# EPUB 3.3 Explainer
+
+**Written**: 2021-02-22, **Updated**: 2021-04-29, Duke Ellington’s birthday. 
+
+## Authors:
+
+ * Dave Cramer (Invited Expert)
+ 
+## Participate
+
+ * [Issue Tracker](https://github.com/w3c/epub-specs/issues)
+ * [Mailing List](https://lists.w3.org/Archives/Public/public-epub-wg/)
+ 
+## Table of Contents
+
+TK
+ 
+## Introduction
+
+**EPUB 3.3** is a packaging mechanism for web content, designed for electronic books. EPUB's roots go back to 1999, with the OEB 1.0 specification from the Open Ebook Forum (which evolved into the IDPF) In 2007, a packaging format was introduced and OEB became EPUB 2.0. In 2011, EPUB 3.0 added support for XHTML5. Changes since then have been minor. The IDPF was absorbed into W3C in 2017, and responsibility for the maintenance of EPUB fell to the EPUB 3 Community Group, which released EPUB 3.2 in 2019 as a community group note. EPUB is twenty-one years old, but has never gone through the W3C Recommendation track, until now. 
+
+EPUB 3.3 includes the following specifications:
+
+* [EPUB 3.3 Overview](https://w3c.github.io/epub-specs/epub33/overview/), a non-normative overview of EPUB 3.3
+* [EPUB 3.3 Core](https://w3c.github.io/epub-specs/epub33/core/), the specification of the file format
+* [EPUB 3.3 Reading Systems](https://w3c.github.io/epub-specs/epub33/rs/), the specification for user agents, known as EPUB Reading Systems. 
+* [EPUB Accessibility 1.1](https://w3c.github.io/epub-specs/epub33/a11y/)
+* [EPUB Accessibility Techniques 1.1](https://w3c.github.io/epub-specs/epub33/a11y-tech/)
+
```

Comment:
Note that the techniques is non-normative. If we list it here then we should probably add the multiple rendition document, too, or list only the rec-track documents.

Reply:
This was meant as a convenience for people who are looking at the questionnaire. I've already made a change indicating that a11y techniques is non-normative. If we remove all non-normative documents we lose the EPUB Overview. I am opposed to adding the multiple rendition note, as that is far outside the core of EPUB.

Reply:
Consistency is not the only virtue. I was using my editorial judgment in deciding what links would be most useful to a reviewer. In my opinion, providing a link to the multiple renditions document would merely serve to muddy the waters. Multiple Renditions are not a thing!

But I do not wish to spend much more time on this issue. And so I will add a link to the multiple renditions document. Should I also add a link to EPUB-CFI? Which, of course, would remind everyone of an EPUB technology that is unrelated to the web...

Reply:
You are right about MR being... well, not really a thing:-)

For the time being we do not have a WG resolution to really publish epubcfi. It is "only" on our repo and the decision whether it will become a WG note is way down the line. So no, I would not put it there.